plugin-customcommand/lxqtcustomcommandconfiguration.cpp

```diff
@@ -113,7 +112,6 @@ LXQtCustomCommandConfiguration::LXQtCustomCommandConfiguration(PluginSettings *s
         HELP_STRUC_DESC6,
         HELP_STRUC_DESC7,
         HELP_STRUC_DESC8,
-        HELP_STRUC_DESC9,
```

Comment:
I didn't check the removal of `%18`, but this removal breaks the string; doesn't it? There's no `%24` after it.